engine/function/src/templates/Numeric.ftl

```diff
@@ -1579,7 +1579,13 @@ public class Numeric {
         result[0] = values[0];
 
         for (int i = 1; i < values.length; i++) {
-            result[i] = compare(result[i-1], values[i]) > 0 ? result[i-1] : values[i];
+            if (isNull(result[i - 1])) {
+                result[i] = values[i];
+            } else if (isNull(values[i])) {
+                result[i] = result[i - 1];
+            } else {
+                result[i] = (${pt.primitive})Math.max(result[i - 1],  values[i]);
```

Comment:
Can we remove the extra space before values[i] at the end

Reply:
```suggestion
                result[i] = (${pt.primitive})Math.max(result[i - 1], values[i]);
```